projects/hmcts-annotation-ui-lib/src/lib/document-viewer/document-viewer.component.spec.ts

```diff
@@ -0,0 +1,145 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {DocumentViewerComponent} from './document-viewer.component';
+import {HttpClientTestingModule} from '@angular/common/http/testing';
+import {DebugElement, SimpleChange} from '@angular/core';
+import {DocumentViewerService} from './document-viewer.service';
+import {of} from 'rxjs';
+import { TransferState } from '@angular/platform-browser';
+import { HmctsEmViewerUiModule } from '../hmcts-em-viewer-ui.module';
+import { EmLoggerService } from '../logging/em-logger.service';
+
+const originalUrl = 'http://api-gateway.dm.com/documents/1234-1234-1234';
+const url = '/demproxy/dm/documents/1234-1234-1234';
+
+class MockTransferState {
+    hasKey() {}
+    remove() {}
+    set() {}
+}
+
+describe('DocumentViewerComponent', () => {
+    const mockTransferState = new MockTransferState();
+    let component: DocumentViewerComponent;
+    let fixture: ComponentFixture<DocumentViewerComponent>;
+    let element: DebugElement;
+    let mockDocuments;
+
+    const DocumentViewerServiceMock = {
+        fetch: () => {
+            return of(mockDocuments);
+        }
+    };
+
+    beforeEach(async(() => {
+        const testingModule = TestBed.configureTestingModule({
+            imports: [HmctsEmViewerUiModule, HttpClientTestingModule],
+            providers: [
+                EmLoggerService,
+                { provide: TransferState, useFactory: () => mockTransferState},
+                { provide: DocumentViewerService, useValue: DocumentViewerServiceMock
+            }]
```

Comment:
maybe best to move the "}" to the line above.

Reply:
fixed